outputs.tf

```diff
@@ -57,3 +57,10 @@ output "target_group_names" {
   description = "Name of the target group. Useful for passing to your CodeDeploy Deployment Group."
   value       = aws_lb_target_group.main.*.name
 }
+
+output "target_group_attachments" {
+  description = "ARNs of the target group attachment IDs."
+  value = [
```

Comment:
I think it would be better to have this as a `map` so that users can identify both target_group (as a key) and a list of attachments (as a value)